script/Ciel/Day 5/05_01_CIEL05_2_2.txt

```diff
@@ -657,32 +706,37 @@ Dumbfounded, I just nod.
 [sha:c4901454fa62096556923d2cfd3bde28db7c295a]{
 -- Page 350, Offset 11796.
 -- 先輩は、俺の知っている先輩だった。
-She's just like the Senpai I know.
+She's the exact same Senpai I've always known.
 }
 [sha:b205984329f183b05b8095dbafe2dc75edaff2d2]{
 -- Page 350, Offset 11797.
 -- こんな地獄の渦中にいながら、あの夜、俺を救ってくれた先輩のままだった。
-In the midst of this hell, she's just like the Senpai that helped me on that night.
+Even in the midst of this hell, she's still the same person that saved me that night.
 }
 [sha:502e11b093e84b8c2b7f5d8931996aadae2be7f7]{
 -- Page 351, Offset 11798.
 -- それが嬉しい反面、不安になる。
-This made me happy, but also uneasy.
+I'm glad, but at the same time, it makes me little uneasy.
 }
 [sha:3b4efad73b7bf299c5da10b38d1813755fd0d572]{
 -- Page 351, Offset 11799.
 -- 先輩にとってこれも日常だというのなら、俺は今まで、この人の何を見てきたというのだろう―――
-If %{i}this%{/i} is routine work for her, then what of the person I've known until now―――
+If %{i}this%{/i} is routine work for her,
+ then just how much is there that I still don't I know about this person―――
 }
 [sha:dce46b3a341bbdb4c750052aab934cf4c5c9b50d]{
 -- Page 352, Offset 11800.
 -- 「そろそろですね。……並の魔術師なら一撃で底をつく火力を２分強持続……只の死徒とは思えない呪力ですが、それもここまででしょう。これ以上はさすがに―――」
-"This should be it... Maintaining this much firepower for more than two minutes, when the average mage would be exhausted after a single hit... I wouldn't have expected such a powerful curse from a common Dead Apostle. But this is the end. Any more shouldn't be―――"
+"Shouldn't be long now...
+ The average mage would be exhausted after loosing a single volley of this intensity,
```

Comment:
losing

Reply:
Pretty sure I mean loosing here - letting loose a volley

Reply:
you're right, my mind kinda wandered off after i saw "exhausted after loosing"

i'd change it to "loosing off"